lib/site_settings/type_supervisor.rb

```diff
@@ -37,7 +37,6 @@ def self.types
       color: 22,
       simple_list: 23,
       emoji_list: 24,
-      html: 25
```

Comment:
If this type is no longer valid, we need a migration to clean it up on the database.

Reply:
I added the migration and renamed the type to `html_deprecated` just to be safe. Could you please take another look?